src/server/gc_worker/gc_worker.rs

```diff
@@ -731,15 +742,20 @@ where
         Ok(lock_infos)
     }
 
-    fn update_statistics_metrics(&mut self) {
-        let stats = mem::take(&mut self.stats);
-
-        for (cf, details) in stats.details_enum().iter() {
-            for (tag, count) in details.iter() {
-                GC_KEYS_COUNTER_STATIC
-                    .get(*cf)
-                    .get(*tag)
-                    .inc_by(*count as u64);
+    fn update_statistics_metrics(&mut self, key_mode: GcKeyMode) {
+        let stats_map = self.stats_map.get(&key_mode);
```

Comment:
We should "take" the `stats_map[key_mode]` to set it to zeors. Otherwise the metrics will be accumulated all the time.